modules/ztp-deploying-siteconfig-hub-side-template.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-deploying-siteconfig-hub-side-template_{context}"]
+= Deploying a SiteConfig with hub-side templating
+
+The following is an example `SiteConfig` custom resource (CR) that uses the `spec.clusters.siteConfigMap.data` field to specify per-site data to be used in hub side templating.
+
+.Example {sno} `SiteConfig` CR with hub-side templating
+[source,yaml]
+----
+include::snippets/ztp_example-siteconfig-hub-side-templating.yaml[]
+----
+[NOTE]
+====
+The generated `ConfigMap` CRs follow the naming convention `ztp-site-<cluster_name>-configMap` and are created under the same namespace as the policy resource.
```

Comment:
Maybe just leave
> The generated `ConfigMap` CRs follow the naming convention `ztp-site-<cluster_name>`.
since you're explaining the namespace aspect below?

Reply:
ok. I removed reference to the namespace and left it at `ztp-site-<cluster_name>-configMap`.

Reply:
It should be `ztp-site-<cluster-name>` though.